concepticondata/conceptlists/Lai-2023-291.tsv

```diff
@@ -0,0 +1,292 @@
+Lai-2023-291-242	103	inside	1606	INSIDE	715	inside
+Lai-2023-291-243	195	the son	1620	SON	220	the son
+Lai-2023-291-244	98	hundred	1634	HUNDRED	824	hundred
+Lai-2023-291-245	179	short	1645	SHORT	973	short
+Lai-2023-291-246	320	the girl	1646	GIRL		the girl
+Lai-2023-291-247	331	the lightning	1654	LIGHTNING		the lightning
+Lai-2023-291-248	184	six	1703	SIX	802	six
+Lai-2023-291-249	175	seven	1704	SEVEN	803	seven
+Lai-2023-291-250	48	eight	1705	EIGHT	804	eight
+Lai-2023-291-251	225	twenty	1710	TWENTY	816	twenty
+Lai-2023-291-252	28	correct (right)	1725	CORRECT (RIGHT)	1045	correct (right)
+Lai-2023-291-253	231	wet	1726	WET	1029	wet
+Lai-2023-291-254	281	rotten (of fruits)	1728	ROTTEN		rotten (of fruits)
+Lai-2023-291-255	186	the sky	1732	SKY	1	the sky
+Lai-2023-291-256	157	the rainbow	1733	RAINBOW	11	the rainbow
+Lai-2023-291-257	1	above	1741	ABOVE	731	above
+Lai-2023-291-258	374	to choose	1750	CHOOSE		to choose
+Lai-2023-291-259	289	stone (of fruit)	1762	STONE (OF FRUIT)		stone (of fruit)
+Lai-2023-291-260	243	the wood (material)	1803	WOOD	511	the wood (material)
+Lai-2023-291-261	30	to cry (weep)	1839	CRY	1485	to cry (weep)
+Lai-2023-291-262	286	sick	1847	SICK		sick
+Lai-2023-291-263	196	the sparrow	1854	SPARROW	336	the sparrow
+Lai-2023-291-264	194	soft	1856	SOFT	1035	soft
+Lai-2023-291-265	22	to buy	1869	BUY	1516	to buy
+Lai-2023-291-266	252	before	1882	BEFORE		before
+Lai-2023-291-267	82	hard	1884	HARD	1034	hard
+Lai-2023-291-268	4	the armpit	1886	ARMPIT	92	the armpit
+Lai-2023-291-269	420	to tie (together)	1917	TIE		to tie (together)
+Lai-2023-291-270	6	the bamboo	1927	BAMBOO	389	the bamboo
+Lai-2023-291-271	209	there	1937	THERE	950	there
+Lai-2023-291-272	139	near	1942	NEAR	975	near
+Lai-2023-291-273	33	the dew	1977	DEW	15	the dew
+Lai-2023-291-274	326	the intestines	1991	INTESTINES		the intestines
+Lai-2023-291-275	60	to flow	2003	FLOW	1502	to flow
+Lai-2023-291-276	37	the dog	2009	DOG	289	the dog
+Lai-2023-291-277	160	the rice plant	2026	RICE PLANT	439	the rice plant
+Lai-2023-291-278	71	the frost	2034	FROST	14	the frost
+Lai-2023-291-279	303	the child	2099	CHILD		the child
+Lai-2023-291-280	123	the man (male human)	2106	MALE PERSON	173	the man (male human) // male
+Lai-2023-291-281	147	old (of person)	2112	OLD (AGED)	1058	old (of person)
+Lai-2023-291-282	136	the nail (fingernail or claw)	2128	CLAW OR NAIL	113	the nail (fingernail or claw)
+Lai-2023-291-283	133	the mouse or rat	2139	MUROID (MOUSE OR RAT)	320	the mouse or rat
+Lai-2023-291-284	125	to marry (a man marries a woman)	2164	MARRY (AS MAN)	1578	to marry (a man marries a woman)
+Lai-2023-291-285	70	the front (front side)	2194	FRONT (PART)	712	the front (front side)
+Lai-2023-291-286	365	thin (object)	2249	THIN (OF SHAPE OF OBJECT)		thin (object)
+Lai-2023-291-287	45	the earthworm	2350	EARTHWORM	363	the earthworm
+Lai-2023-291-288	38	the dream	2374	DREAM	699	the dream
+Lai-2023-291-289	93	horizontal	2376	HORIZONTAL	1001	horizontal
+Lai-2023-291-290	430	wide	2531	BROAD		wide
```

Comment:
Perhaps, [WIDE](https://concepticon.clld.org/parameters/1243) or [BROAD OR WIDE](https://concepticon.clld.org/parameters/2652) is a more suitable mapping?